scripts/exhafs_grid.sh

```diff
@@ -82,6 +82,8 @@ elif [ $gtype = nest ] || [ $gtype = regional ]; then
   export jstart_nest=${jstart_nest:-238}
   export iend_nest=${iend_nest:-1485}
   export jend_nest=${jend_nest:-1287}
+  export add_comp_size_i=${add_comp_size_i:-400}
```

Comment:
@BantwaleEnyew-NOAA Could you please revert these changes, which should not be part of this PR?